cloudflare.install

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @file
+ * Install, update, and uninstall functions for the CloudFlare module.
+ */
+
+use Drupal\Core\Url;
+
+/**
+ * Implements hook_requirements().
+ */
+function cloudflare_requirements($phase) {
+  $requirements = [];
+  if ($phase == 'install') {
+    if (!class_exists('\CloudFlarePhpSdk\ApiEndpoints\ZoneApi')) {
+      $requirements['cloudflare_library'] = [
+        'title' => t('CloudFlare - Php Sdk'),
+        'description' => t('CloudFlare requires the d8-contrib-modules/cloudflarephpsdk library.'),
+        'severity' => REQUIREMENT_ERROR,
```

Comment:
First thing that pops to my mind: is there a check for Purge too? Can it run without the SDK (and thus crash)?